.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.6']
+        julia-version: ['1']
```

Comment:
For context: https://github.com/JuliaLang/Example.jl/pull/68#discussion_r995947019